blog/2023-11-01-feature-flags-sdks-architectures.md

```diff
@@ -0,0 +1,161 @@
+---
+slug: 'feature-flags-sdks-architectures'
+title: 'Feature Flags services Server-side SDKs different architectures approaches'
+date: 2023-11-01
+authors: [liran2000]
+description: "Feature Flags services Server-side SDKs different architectures approaches"
+tags: [client, sdk]
+draft: false
+---
+
+Server-side SDKs (software development kits) are common for letting applications like microservices interact with
+Feature Flags services.  
+Feature Flags SDKs have several used functionalities, mainly feature flag evaluation with or without a context. Each
+Feature Flags service can publish SDKs in multiple programming languages.  
+The Feature Flags services commonly expose APIs via endpoints like REST HTTP and/or gRpc.
+SDKs are an important layer, as network traffic loads affects both the application and the Feature Flags services.
+When the Feature Flags service is a Cloud SaaS service, some services support a relay proxy or a sidecar application
+which can be deployed on an organization network. The
+Relay Proxy lets multiple clients to connect to a local proxy, reducing number of outbound connections to the Cloud
+Service. Considering that a large amount of microservices using SDKs instances can be deployed, this can be significant.
+
+
+<!--truncate-->
+
+## Feature Flags services Server-side SDKs different architectures approaches
```

Comment:
```suggestion
## Different approaches for architecture server-side SDKs
```

Reply:
"Different approaches for server-side SDKs architecture" would be ok ?

Reply:
Sure, sounds good!